tests/test_client.py

```diff
@@ -449,7 +449,7 @@ def test_call_rpc(self):
         self._mock_http(d, status=(504, "gateway timeout"))
         self.assertRaises(api.ProtocolError, self.sg._call_rpc, "list", a)
         self.assertEqual(
-            4,
+            3,
```

Comment:
what's this change?

Reply:
I standardize the number of retries to be reused on the library.

Reply:
Can you use `self.sgMAX_ATTEMPTS` instead or does it make sense to keep these 3 everywhere?

Reply:
I think it's feasible.